dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/EditDatasetMetadataPage.java

```diff
@@ -160,6 +141,25 @@ private String returnToLatestVersion() {
         return "/dataset.xhtml?persistentId=" + dataset.getGlobalIdString() + "&version=" + workingVersion.getFriendlyVersionNumber() + "&faces-redirect=true";
     }
 
+    private void handleUpdateDatasetExceptions(Throwable throwable) {
+        if (throwable instanceof EJBException) {
```

Comment:
CommandException no, ValidationException yes since it is from javax package which we don't have controll over. Maybe I should just use our ValidationException then, since I see now that we have it.

Reply:
So have you changed anything here?

Reply:
I saw that Dataverse ValidationException was used somewhere else and it would require additional refactoring, since it would require changing logic regarding catching exceptions in other places, so i thought that keeping it like that is also fine IMO.